src/main/java/org/example/bookstore/controller/BookControllerImpl.java

```diff
@@ -35,4 +40,15 @@ public BookDto getBookById(@PathVariable Long id) {
     public BookDto createBook(@RequestBody CreateBookRequestDto requestDto) {
         return bookService.save(requestDto);
     }
+
+    @PutMapping("/{id}")
+    public Book updateBookById(@PathVariable Long id,@RequestBody BookDto book) {
+        return bookService.updateBookById(id,book);
+    }
```

Comment:
in this method better to `return BookDto` and one of receiving parameters type must be `CreateBookRequestDto` like in previous methods